auth/client/src/main/resources/schema/elytron-client-1_4.xsd

```diff
@@ -798,10 +802,16 @@
                 An OAuth 2 bearer token.
             </xsd:documentation>
         </xsd:annotation>
-        <xsd:choice maxOccurs="unbounded">
-            <xsd:element name="client-credentials" type="oauth2-client-credentials-type" minOccurs="0" maxOccurs="1"/>
-            <xsd:element name="resource-owner-credentials" type="resource-owner-credentials-type" minOccurs="0" maxOccurs="1"/>
-        </xsd:choice>
+        <xsd:sequence>
```

Comment:
Since this originally was a ```choice``` with ```maxOccurs``` set to ```unbounded```, this allowed for the choice between ```client-credentials``` and ```resource-owner-credentials``` to be done any number of times, which results in being able to specify them in any order, any number of times. I think we just need to update that existing choice to also allow the two new elements - so any of these four elements can be specified any number of times and in any order.

Reply:
Fixed